schema/ethernet.yml

```diff
@@ -168,6 +168,14 @@ properties:
         type: integer
         minimum: 1
         maximum: 4094
+      mac-address-bypass:
+        description: Trigger mac-auth when a new ARP is learned.
+        type: boolean
+      mac-address-bypass-timout:
+        description: Trigger mac-auth when a new ARP is learned.
```

Comment:
(minor comment)
Also, shouldn't this description be a bit different from the field above?
Since it's a timeout value, not an actual option to <trigger mac-auth...>

Reply:
This was a typo. Will correct.